docs-en/chapter_data_structure/basic_data_types.md

```diff
@@ -31,36 +31,36 @@ The following table lists the space occupied, value range, and default values of
 | Char    | `char`   | 2 bytes        | 0                        | $2^{16} - 1$            | 0              |
 | Boolean | `bool`   | 1 byte         | $\text{false}$           | $\text{true}$           | $\text{false}$ |
 
-Please note that the above table is specific to Java's fundamental data types. Each programming language has its own data type definitions, and their space occupied, value ranges, and default values may differ.
+Please note that the above table is specific to Java's basic data types. Each programming language has its own data type definitions, and their space occupied, value ranges, and default values may differ.
```

Comment:
"Each programming language has its own data type definitions, and their space occupied, value ranges, and default values may differ."
can be changed to ->
"Every programming language has its distinct data type definitions, which might differ in space occupied, value ranges, and default values."